tests/integration/targets/ssh_config/tasks/main.yml

```diff
@@ -186,6 +186,25 @@
       - not mut_ex.changed
       - "'parameters are mutually exclusive' in mut_ex.msg"
 
+- name: Check if proxycommand and proxyjump are mutually exclusive
+  community.general.ssh_config:
+    ssh_config_file: "{{ ssh_config_test }}"
+    host: "example.com"
+    hostname: github.com
+    proxycommand: "ssh jumphost.example.com -W %h:%p"
+    proxyjump: "jumphost.example.com"
+    identity_file: '{{ ssh_private_key }}'
+    port: '2224'
+    state: present
+  register: proxy_mut_ex
+  ignore_errors: true
+
+- name: Check mutual exclusive test - proxycommand and proxyjump
+  assert:
+    that:
+      - not proxy_mut_ex.changed
+      - "'parameters are mutually exclusive' in mut_ex.msg"
```

Comment:
It would be nice to have an actual test of `jumpproxy` and not just one of AnsibleModule's `mutually_exclusive` validation.

Reply:
Yes, I did think test were a bit sparse.. Would you like me to clone the existing tests used for proxycommand and adjust for proxy jump?

Reply:
It would be great if you could do that!

> Apologies for this - new to contribution to ansible.

No need to apologize! Thanks again for contributing :)

Reply:
Added further test in commit : https://github.com/ansible-collections/community.general/pull/6205/commits/b1194aba46809aca9b9dfd046898ffb06e269bdf

Local execution looks good

```
PLAY RECAP *********************************************************************
testhost                   : ok=94   changed=23   unreachable=0    failed=0    skipped=1    rescued=0    ignored=3   
```